### PR TITLE
Fix setup instructions

### PR DESCRIPTION
**Pull request by SebastianWagner2**
bruh

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,10 +21,11 @@ After making sure that you have all the required dependencies take the following
 2. Clone this repository by running the command `git clone https://github.com/HackHPI/meme-challenge.git`.
 3. Open the project in your prefered code editor.
 4. In the project's root directory, run `yarn` to fetch all project dependencies.
-5. Run `yarn dev` (it's gonna fail we suck).
+5. In the `meme-challenge` directory, create a file `.env` identical to the [exemplary version](.env.example) (`.env.example`). This file must contain all necessary variable information about the environment the project should be run in. We will set the missing variables in a later step.
```
Comment (Benjamin-Frost) [inline README.md:24]: Doesn't have to be named this way. Devs can rename the folder to anything they like:

```suggestion
5. In the project's root directory, create a file `.env` identical to the [exemplary version](.env.example) (`.env.example`). This file must contain all necessary variable information about the environment the project should be run in. We will set the missing variables in a later step.
```

File: README.md [hunk 2]
```diff
@@ -41,11 +42,11 @@ After making sure that you have all the required dependencies take the following
...
      - Go to `Credentials` tab and set some password (e.g. `123`). Also, prefer deselecting the `Temporary` option, as it will prompt you to set a new password. This will be great for the deployed version but unnecessary for local development.
    - Lastly, we need to prepare something for further setup. When you start the project locally, it needs to know what realm and what client to use in the environment it was just started in. To save some time finding this information later on, navigate to `Clients`, click on yours, go to the `Credentials` tab and copy the `Client secret`. If you regenerate this, you need to update it in the code as well.
-8. In the `meme-challenge` directory, create a file `.env` identical to the [exemplary version](.env.example) (`.env.example`). This file must contain all necessary variable information about the environment the project should be run in. Scroll to the bottom of the file to find the empty variables that need to be set.
+9. In the `meme-challenge` directory, find the `.env` file you've created in step 5. Scroll to the bottom of the file to find the empty variables that need to be set.
```
Comment (Benjamin-Frost) [inline README.md:45]:
```suggestion
9. In the project's root directory, find the `.env` file you've created in step 5. Scroll to the bottom of the file to find the empty variables that need to be set.
```

**Review by benjamin-frost — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (benjamin-frost) [inline README.md]:
```suggestion
5. In the project's root directory, create a file `.env` identical to the [exemplary version](.env.example) (`.env.example`). This file must contain all necessary variable information about the environment the project should be run in. We will set the missing variables in a later step.
```

File: README.md (hunk 2 above)
Comment (benjamin-frost) [inline README.md]:
```suggestion
9. In the project's root directory, find the `.env` file you've created in step 5. Scroll to the bottom of the file to find the empty variables that need to be set.
```

**Review by benjamin-frost — CHANGES_REQUESTED** [review]

**Review by benjamin-frost — APPROVED** [review]

**Pull request merged**